client/pkg/pynuodb.py

```diff
@@ -25,24 +25,15 @@ def __init__(self):
         self.stage = self.staged[0]
 
     def download(self):
-        # Find the latest release
-        pypi = PyPIMetadata(self.__PKGNAME)
-
-        self.setversion(pypi.version)
-
-        self._file = Artifact(self.name, 'pynuodb.tar.gz',
-                              pypi.pkgurl, chksum=pypi.pkgchksum)
-        self._file.update()
+        pass
```

Comment:
You can just remove this method if it doesn't do anything.  The superclass already provides dummy methods for all the steps.